### DISCUSSION TITLE
Disabled Link Component

### DISCUSSION THREAD

**Discussion by HelloVVolf** [inertiajs/inertia#1447] [Help]
How to disable Link components? I've added `<Link disabled={ true }/>` but it's not work.

**Comment by ldesperrois** [comment]
I have the same question of you @HelloVVolf

**Reply by jankapunkt** [reply]
Here, too. The generated link seems to be disabled, yet it fires a get request that partially reloads the current page.

Current workaround on my end is to add css  `pointer-events: none` (or with tailwind) when in disabled is true but that has its own downsides.
